sap/5.7/modules/ROOT/pages/sap-connector-config-snc-kerberos.adoc

```diff
@@ -0,0 +1,149 @@
+= Configuring an SAP Secure Network Communication Connection with Kerberos
+
+Configure an SAP Secure Network Communication (SNC) with Kerberos to take advantage of security features such as secure data communication between the SAP system client and server, application-level end-to-end security, and the ability to change security products without impacting your SAP business applications. Enabling SNC with Kerberos requires you to configure the client environment, the SAP server, and Anypoint Connector for SAP (SAP Connector). 
+
+NOTE: SAP does not provide support for third-party solutions.
+
+== Before You Begin
+
+To configure an SAP SNC with Kerberos, you must have access to:
+
+* The SAP GUI application. To install this client, Java must already be installed on your machine. Then, download the installer for your operating system from SAP.
+* An SAP ECC instance that can use SNC system variables
+* The following transactions:
+** `STRUST` (Trust Manager)
+** `SM30` (Table Maintenance)
+** `SNC1` (Generate SNC name for user)
+** `SU01` (User Maintenance)
+** `SM59` (RFC Destinations)
+
+== Configure Active Directory
+
+Create a new Service Principal Name (SPN) according to the operating system you are using and export the keytab.
+
+If you are using Windows, follow these steps:
+
+. Create a service user `adminsap` in Active Directory.
+. Check *Password never expires* and uncheck *Use DES encryption types for this account*.
+. Create a Service Principal Name (SPN) `SAPService/sapdev@SSODOMAIN.LAB` for this user in Active Directory.
++
+`setspn -A SAPService/sapdev@SSODOMAIN.LAB DOMAIN_SHORT\sidadm`
++
+. Export the keytab for this SPN from Active Directory. The password is the password to set for the keytab and the SID is the file name of the keytab.
++
+`ktpass -princ SAPService/sapdev@SSODOMAIN.LAB -mapuser DOMAINSHORT\sidadm -crypto AES256-SHA1 -ptype KRB5_NT_PRINCIPAL -mapop set -pass <*password*> -out <SID>aes.keytab`
+
+== Configure the SAP Server
+
+Change the domain and the KDC in the operating system of the SAP Server to set up Kerberos, and add the parameters and the library that authorize you to use SNC:
```

Comment:
Same comment as before. (I think it's great to summarize what's involved in the intro, but make sure it's obviously a summary.)

Reply:
I'm confused by this sentence.
Is it: "To set up Kerberos, do this..."

Reply:
Should Server be lc?